core-java/src/main/java/org/baeldung/equalshashcode/entities/ComplexClass.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.List;
 import java.util.Set;
-
+// all classes in package moved to core-java-lang as part of com.baeldung.* package
```

Comment:
As you can see here: https://github.com/eugenp/tutorials/pull/5649/files#diff-ff9a246bf080d2677d59ed9e474a87e9L1
The code for this particular article was duplicated (in com.baeldung). I will delete the code on both packages after successfully migrating the code to the new module (y)